src/test/resources/com/puppycrawl/tools/checkstyle/checker/InputCheckerTestCharset.java

```diff
@@ -1,5 +1,15 @@
+/*xml
+<module name="Checker">
+  <module name="com.puppycrawl.tools.checkstyle.checks.sizes.LineLengthCheck">
+    <property name="max" value="75"/>
+  </module>
+  <property name="charset" value="IBM1098"/>
+</module>
+*/
+
```

Comment:
Put on this line, line 9 ,  `// violation 6 lines above`.

Reply:
Done